tests/unit/test_rust_backend.py

```
@@ -0,0 +1,519 @@
+"""Comprehensive unit tests for rust_backend module."""
+import pytest
+import unittest.mock as mock
```

Ruff (0.11.9)

Use `from unittest import mock` in lieu of alias

Replace with `from unittest import mock`

(PLR0402)